.github/workflows/pr.yaml

```diff
@@ -203,7 +238,7 @@ jobs:
     - name: lintr
       run: |
         library(lintr)
-        linters <- with_defaults(line_length_linter = NULL, cyclocomp_linter = NULL)
+        linters <- with_defaults(line_length_linter = NULL, object_usage_linter = NULL)
```

Comment:
Should we keep `cyclocomp_linter = NULL` (added in commit 4e17a8df806c4f37c83874ebf46b97c924586a5a )?

Reply:
Probably yes and we should also add it to the IUC workflow, or?

Reply:
I suppose it would make life easier on tools-iuc too.

Reply:
done. and https://github.com/galaxyproject/tools-iuc/pull/3398